### PR TITLE
Implement Dark Mode Support

### LINKED ISSUE #17: feat: Implement Dark Mode Support

### PR DESCRIPTION
**Pull request by ruru-m07**
# this pull request close #17

## after this i see this UI

Screenshot 2024-04-12 035411

## - this is too bad i make it batter now.

- [x] dark theme
Screenshot 2024-04-12 041140

- [x] light theme
Screenshot 2024-04-12 041129

### CODE REVIEW

**Review by ruru-m07 — COMMENTED** [review]
double check changes 👍

**Comment by ruru-m07** [comment]
# checks ✅

- [x] 1. type check
- [x] 2. next lint
- [x] 3. next build

----

# 1. ✅
image

# 2. ✅
image

# 3. ✅
image

**Comment by ruru-m07** [comment]
# checks ✅
* [x]  1. type check ✅
* [x]  2. next lint ✅
* [x]  3. next build ✅

**Comment by ruru-m07** [comment]
@jeremyphilemon can you review this pull request?

**Review by athrael-soju — APPROVED** [review]
This is great work. I've tested it and it works just fine. @jeremyphilemon can we get this reviewed please?

File: components/chat-panel.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ export function ChatPanel({
               <div
                 key={example.heading}
                 className={cn(
-                  'cursor-pointer bg-zinc-50 text-zinc-950 rounded-2xl p-4 sm:p-6 hover:bg-zinc-100 transition-colors',
+                  'cursor-pointer bg-secondary rounded-2xl p-4 sm:p-6 transition-colors',
```
Comment (zvictor) [inline components/chat-panel.tsx:66]:
```suggestion
                  'cursor-pointer bg-background rounded-2xl p-4 sm:p-6 transition-colors',
```

you are removing the colour distinction of the button if not using something more distinct such as bg-background

**Pull request closed without merging**